From 58499e1fac35c15529a15c7ebcd10234b50f612f Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 9 Sep 2022 16:42:51 +0200
Subject: [PATCH 77/84] arm64: dts: rockchip: rk3399-pinephone-pro: Enable POGO
 pins I2C

So that userspace apps can use it.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
(cherry picked from commit a91c41c0dbfacd256b8beeafe28c0bfaa3202166)
---
 .../boot/dts/rockchip/rk3399-pinephone-pro.dts     | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index fd1c04c3868a..bbc56d8d0c5b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -986,6 +986,14 @@ af8133j: compass@1c {
 	};
 };
 
+/* This bus is exposed on POGO pins */
+&i2c5 {
+	i2c-scl-rising-time-ns = <450>;
+	i2c-scl-falling-time-ns = <15>;
+	pinctrl-0 = <&i2c5_xfer &pogo_int_pin>;
+	status = "okay";
+};
+
 &i2s0 {
 	rockchip,playback-channels = <2>;
 	rockchip,capture-channels = <2>;
@@ -1203,6 +1211,12 @@ vsel2_pin: vsel2-pin {
 		};
 	};
 
+	pogo {
+		pogo_int_pin: pogo-int {
+			rockchip,pins = <3 RK_PA0 RK_FUNC_GPIO &pcfg_pull_up>;
+		};
+	};
+
 	sdio-pwrseq {
 		wifi_enable_h_pin: wifi-enable-h-pin {
 			rockchip,pins = <0 RK_PB2 RK_FUNC_GPIO &pcfg_pull_none>;
-- 
2.55.0

